### DISCUSSION TITLE
Markdown/HTML injection

### DISCUSSION THREAD

**Discussion by numpde** [idom-team/idom#880] [Question] [answered]
I'd like to display some Markdown in a component; kinda like the window I'm looking at right now. Suppose I convert that to plain HTML first. Should I use `html.iframe` or is there a good/better alternative? Thank you.

**Comment by Archmonger** [comment, marked as answer]
You can use markdown with the following steps:
1) Convert the markdown to a string using your preferred library (such as [python-markdown](https://pypi.org/project/Markdown/))
2) Use [`idom.utils.html_to_vdom`](https://idom-docs.herokuapp.com/docs/_auto/apis.html#idom.utils.html_to_vdom) to convert it to VDOM
3) Render the VDOM within an IDOM component.

```python
import markdown
from idom.utils import html_to_vdom
from idom import component, run, html

example_md = "### Hello World"
example_html = markdown.markdown(example_md)
example_vdom = html_to_vdom(example_html)

@component
def example_component():
    return html.div(example_vdom)


run(example_component)
```

**Comment by numpde** [comment]
Thank you, that's the one.
